docs/source/slider.rst

```diff
@@ -0,0 +1,14 @@
+******
+Slider
+******
+
+.. currentmodule:: seirmo
+
+The models provide a framework for the implementation of any models.
+
+Overview:
+
+- :class:`_SliderComponent`
```

Comment:
If we register the SliderComponent it is not longer private, so it would make more sense if we didn't prepend `_`, i.e. can you change the name to `SliderComponent`

Reply:
So should we register SliderComponent? Or keep it private?